src/aruco_pose/src/redis_insert.py

```diff
@@ -0,0 +1,16 @@
+"""Copyright 2023 Brookhaven National Laboratory BSD 3 Clause License. See LICENSE.txt for details."""
+
+import redis
+
+# Connect to Redis
+# Note: Change the host IP to map your server.
+client = redis.Redis(host="192.168.56.1", port=6379, db=0)
```

Comment:
I know this is just a tool for sanity checking, but perhaps we could do this more programmatically with:

```bash
podman container inspect -f '{{.NetworkSettings.IPAddress}}' redis-container
```

Reply:
A note is added to retrieve the IP address
https://github.com/maffettone/erobs/blob/281d3dfdb27f2387ff8b83695a9f3b0b94f6624b/src/aruco_pose/src/redis_insert.py#L6-L10